buttons/src/main/java/com/decathlon/vitamin/compose/buttons/VitaminButtonsSizes.kt

```diff
@@ -24,7 +24,7 @@ object VitaminButtonsSizes {
         fontSize: TextStyle = VitaminTheme.typography.button
             .copy(fontSize = VitaminTheme.typography.text1.fontSize),
         minWidth: Dp = 84.dp,
-        height: Dp = 64.dp,
+        height: Dp = 56.dp,
```

Comment:
Did you change & verify paparazzi tests ?

Reply:
Done